test/net/sourceforge/kolmafia/ModifiersTest.java

```diff
@@ -998,4 +999,26 @@ public void correctlyAppliesAmphibianSympathyToDodecapede(
   public void identifiesVariableModifiers(String skillName, boolean variable) {
     assertThat(Modifiers.getModifiers("Skill", skillName).variable, equalTo(variable));
   }
+
+  @ParameterizedTest
+  @CsvSource({
+    FamiliarPool.VAMPIRE_VINTNER + ", " + Modifiers.BOOZEDROP + ", 1.5",
+    FamiliarPool.PEPPERMINT_RHINO + ", " + Modifiers.CANDYDROP + ", 1.0",
```

Comment:
can we have another test confirming the rhino is a 1x fairy (for straight itemdrop), please?

Reply:
done and the test failed! good spot

Reply:
Well, the test didn't fail, the test failed for the vintner and cbb